### ISSUE TITLE
Removing weird `is...` getters?

### ISSUE THREAD

**Issue by danielgindi** [danielgindi/Charts#1752]
When first implementing this library, ObjC compatibility was in mind all the time.
And so, `is` getters were written too. The problem is, there was no feature like `getter=...` in Swift. (There's a relatively new option to use `@objc(is...)` on the getter, but that also requires declaring an additional private variable)

Now it seems that as Swift is getting more mature, there is not going to be such a feature.
Apple also tends to write Swift properties without the `is` prefix, unless there's only a getter.  

I want to hear your thoughts, pros and cons, before making a decision on this.
Shoot.

**Comment by danielgindi** [comment]
@petester42

**Comment by petester42** [comment]
I'm all for normailizing the names with swift conventions. In addition you can have a public getter and a private setting for the same property in swift. https://www.natashatherobot.com/swift-magic-public-getter-private-setter/

**Comment by danielgindi** [comment]
I actually didn't understand how would a private getter assist us in this case...

**Comment by petester42** [comment]
No no. The setter would be private so you don't need a separate private property to have only a public getter. So we can use the `@objc` tag on that property to change the name.

**Comment by danielgindi** [comment]
Still I don't get how that would help?
Without private setter, for `drawValuesEnabled`, we would:

```
private var _drawValuesEnabled: Bool = false
open var drawValuesEnabled: Bool
{
  @objc(isDrawValuesEnabled) get { return _drawValuesEnabled }
  set { _drawValuesEnabled = newValue }
}
```

Which will achieve @objc compatibility.

Also I'm not sure about this yet, as Apple did change `hidden` to `isHidden` on [`UIView`](https://developer.apple.com/reference/uikit/uiview/1622585-hidden) in Swift version. Can't find a consistent rule, but I think that just `hidden` sounds more Swifty, and `isHidden` more objc...

**Comment by petester42** [comment]
What I am talking about would look like this: 
```swift
@objc(isDrawValuesEnabled)
open private(set) var drawValuesEnabled: Bool
```

No need for a separate `_drawValuesEnabled`

**Comment by danielgindi** [comment]
If the setter is private then what's the point? We need a public setter...

The private variable is because we do `@objc(isDrawValuesEnabled) get`, we change specifically the name of the getter, not the whole property

**Comment by petester42** [comment]
Ya I get your point now. I think the way apple is pushing is to have it be as readable as possible. So we could just rename `drawValueEnabled` to `canDrawValues` and not worry about the objc stuff.

**Comment by danielgindi** [comment]
Yeah but that's one property, there are lots of properties to worry about, a lot of names.
That is why I thought about just removing the `is` getters completely

**Comment by petester42** [comment]
That sounds good to me.

**Comment by liuxuan30** [comment]
seems swift likes isHidden style.. I checked several places like `isContinuous`, `isKeyWindow`

**Comment by jjatie** [comment]
Swift's [API guidelines](https://swift.org/documentation/api-design-guidelines/) are clear on this,

> Uses of Boolean methods and properties should read as assertions about the receiver when the use is nonmutating, e.g. x.isEmpty, line1.intersects(line2).
